### ISSUE TITLE
Hosting costs bounty

### ISSUE THREAD

**Issue by kkremitzki** [FreeCAD/FreeCAD-Homepage#89]
Creating an issue to track request of payout for hosting costs from November 2020 to April 2022, inclusive. Total cost $802.31.

**Comment by kkremitzki** [comment]
Closing.

**Comment by donovaly** [comment]
Thanks. This is an issue for the FC association -> inviting @yorikvanhavre.

Concerning the costs, what do we pay monthly for what? 44.55 $ is relatively high, maybe we find a cheaper hoster. If this is not for the public, please bring it to the next FC association meeting.

**Comment by bblacey** [comment]
I have used [ssdnodes](https://www.ssdnodes.com) for several years for other projects and they are very cost-effective and performance.  I am in no way affiliated with ssdnodes, just happy with the service the provide for my needs and passing this info along in case it is of use.

**Comment by kkremitzki** [comment]
We are using two $20 DigitalOcean droplets plus automatic backups. The costs for these were covered as part of their open source project sponsorship, but we exceeded the allocation we applied for. We do qualify for a higher tier which does not have an explicit cap, but I need to get in touch with our contact at their open source programs group to get the ball rolling on that, which I plan to do shortly, just wanted to go ahead and process this in a public manner--if it's possible to move the issue, for example to the FC association, that might be a good idea.
